apps/minifront/src/state/shared.ts

```diff
@@ -86,3 +88,11 @@ const groupByAccount = (acc: BalancesByAccount[], curr: BalancesResponse): Balan
 export const balancesByAccountSelector = (
   zQueryState: AbridgedZQueryState<BalancesResponse[]>,
 ): BalancesByAccount[] => zQueryState.data?.reduce(groupByAccount, []) ?? [];
+
+export const filteredBalancesByAccountSelector =
```

Comment:
Hm... is this meant to be used throughout the app? At the moment, it's only being used in the dashboard, so I'd personally prefer to colocate this with the Dashboard component (or at least in a helpers file next to it).

My concern is that, if this is going to be used in multiple places, it's a bit specific — it filters out certain asset types that we might not want to filter out everywhere, and it sorts by priority, which may (?) not be needed everywhere. Thoughts?